administrator/language/en-GB/en-GB.com_cpanel.ini

```
@@ -25,6 +25,8 @@ COM_CPANEL_MSG_HTACCESS_BODY="A change to the default .htaccess and web.config f
 COM_CPANEL_MSG_HTACCESS_TITLE=".htaccess & web.config Update"
 COM_CPANEL_MSG_HTACCESSSVG_TITLE="Additional XSS protection for the usage of SVG files"
 COM_CPANEL_MSG_HTACCESSSVG_BODY="<p>Since 3.9.21 Joomla is shipped with an additional security rule in the default htaccess.txt. This rule will protect users of svg files from potential Cross-Site-Scripting (XSS) vulnerabilities.<br>The security team recommends to manually apply the necessary changes to any existing .htaccess file, as this file can not be updated automatically.</p><p><strong>Changes for .htaccess</strong></p><pre>&lt;FilesMatch \"\.svg$\"&gt;\n  &lt;IfModule mod_headers.c&gt;\n    Header always set Content-Security-Policy \"script-src 'none'\"\n  &lt;/IfModule&gt;\n&lt;/FilesMatch&gt;</pre><p>Currently we are not aware of a method to conditionally configure this on IIS web servers, please contact your hosting provider for further assistance.</p>"
+COM_CPANEL_MSG_INVALID_CREATED_DATES_TITLE="Some of your content doesn't have a valid created date"
```

That's not really a title - it's a message. I would suggest rephrasing to something like

```suggestion
COM_CPANEL_MSG_INVALID_CREATED_DATES_TITLE="Warning - invalid created date"
```